kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectionIndex.vue

```diff
@@ -98,7 +98,16 @@
           :contentNode="channel"
           :to="selectFromChannelsLink(channel)"
           :headingLevel="3"
-        />
+        >
+          <template #belowTitle>
+            <div
+              v-if="contentCardMessage(channel)"
+              style="margin: 0 0 0.5rem"
+            >
+              {{ contentCardMessage(channel) }}
+            </div>
+          </template>
```

Comment:
This text was previously displayed through a KTextTruncator with maxlines=1 - was it intentional not to carry that over here?

Reply:
It was deliberate inasmuch as it was intended to be consistent with how this text was displayed on folder and resource cards - but perhaps they should all be following the truncator pattern instead?

Reply:
I do think being consistent across the cards would be useful. I don't know if I have a strong thought about using the truncator or not. I guess one of the benefits would be consistent spacing, and we probably wouldn't be losing that much information

Reply:
Honestly I don't have too strong of an opinion either pro or against truncation, but I would be in support of consistency across the cards

Reply:
I think simply because we are keenly aware that some translated strings do end up being longer than others, it is probably worth being proactive with the truncator, and we can apply that everywhere we are inserting card messages.